lib/dynbuf.h

```diff
@@ -58,5 +58,5 @@ size_t Curl_dyn_len(const struct dynbuf *s);
 #define DYN_RTSP_REQ_HEADER (64*1024)
 #define DYN_TRAILERS        (64*1024)
 #define DYN_PROXY_CONNECT_HEADERS 16384
-
+#define DYN_QLOG_NAME       1024
```

Comment:
Don't we already have other definitions for this. Or maybe wrappers for platform specific definitions in linux/limits.h e.g PATH_MAX

Reply:
I prefer to set our own fixed limit rather than introducing ifdefs and include for PATH_MAX, which we then won't have control or knowledge exactly how big that is on various platforms.

Reply:
But isn't 1024 a bit too high? On most systems (most unixes) 255 or 256 is the maximum I believe

Reply:
On the contrary, that's actually on the lower side but I still think its fine. See https://eklitzke.org/path-max-is-tricky for a long journey down the PATH_MAX craziness..